File: 01_programming_basics/exercises/02_exercise_conditional_statements/04_toy_shop.py
trip_price = float(input())
number_puzzles = int(input())
number_talking_dolls = int(input())
number_teddy_bears = int(input())
number_minions = int(input())
number_trucks = int(input())
puzzle_price = number_puzzles * 2.60
talking_dolls_price = number_talking_dolls * 3
teddy_bear_price = number_teddy_bears * 4.10
minion_price = number_minions * 8.20
trucks_price = number_trucks * 2
total_numbers = number_puzzles + number_talking_dolls + number_teddy_bears + number_trucks + number_minions
total_sum = puzzle_price + talking_dolls_price + teddy_bear_price + minion_price + trucks_price
if total_numbers >= 50:
    total_sum = total_sum * 0.75
rent = total_sum * 0.10
total = total_sum - rent
difference = abs(total - trip_price)
if total >= trip_price:
    print(f"Yes! {difference:.2f} lv left.")
else:
    print(f"Not enough money! {difference:.2f} lv needed.")

File: 05_JavaScriptBasics/02_Exercises/02_ConditionalStatements/04_toyShop.js
function toyShop(input){
    let puzzlePrice = 2.60
    let talkingDollPrice = 3
    let teddyBearPrice = 4.10
    let minionPrice = 8.20
    let truckPrice = 2

    let tripPrice = Number(input[0])
    let numPuzzles = Number(input[1])
    let numTalkingDolls = Number(input[2])
    let numTeddyBears = Number(input[3])
    let numMinions = Number(input[4])
    let numTrucks = Number(input[5])

    let numToys = numPuzzles + numTalkingDolls + numTeddyBears + numMinions + numTrucks
    let totalPrice = numPuzzles * puzzlePrice + numTalkingDolls * talkingDollPrice + numTeddyBears * teddyBearPrice + numMinions * minionPrice + numTrucks * truckPrice

    if (numToys >= 50){
        totalPrice *= 0.75
    }

    totalPrice *= 0.90

    if (totalPrice >= tripPrice){
        console.log(`Yes! ${(totalPrice - tripPrice).toFixed(2)} lv left.`)
    } else {
        console.log(`Not enough money! ${(tripPrice - totalPrice).toFixed(2)} lv needed.`)
    }
}
